diff --git a/node_modules/carlo/lib/carlo.js b/node_modules/carlo/lib/carlo.js
index 902e137..7a84c8f 100644
--- a/node_modules/carlo/lib/carlo.js
+++ b/node_modules/carlo/lib/carlo.js
@@ -52,8 +52,10 @@ class App extends EventEmitter {
   async init_() {
     debugApp('Configuring browser');
     let page;
+    let pageContext = (await this.browser_.pages())[0]
+
     await Promise.all([
-      this.browser_.target().createCDPSession().then(session => {
+      pageContext.target().createCDPSession().then(session => {
         this.session_ = session;
         if (this.options_.icon)
           this.setIcon(this.options_.icon);
@@ -64,9 +66,7 @@ class App extends EventEmitter {
             'midi',
             'notifications',
             'camera',
-            'microphone',
-            'clipboard-read',
-            'clipboard-write']),
+            'microphone']),
       this.browser_.pages().then(pages => page = pages[0])
     ]);
 
